target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -1238,7 +1250,9 @@ TARGET_DEVICES += CPA-ZNTE60T
 define Device/MAGIC
   $(Device/bcm63xx)
   DEVICE_VENDOR := Telsey
-  DEVICE_MODEL := MAGIC (Alice W-Gate)
+  DEVICE_MODEL := MAGIC
+  DEVICE_ALT0_VENDOR := Alice
+  DEVICE_ALT0_MODEL := W-Gate
```

Comment:
I don't think this was ever sold under the "Telsey Magic" name, as Telsey is an OEM AFAICT. "Alice W-Gate" should be the main title. Magic might be justthe Board ID? 
Googling suggests the Telsey model names sold as Alice W-Gate are CPA-JY-E51TLI and CPA-JY-E53TLI. Maybe. I don't speak Italian, and the model is quite old.

Reply:
@KanjiMonster can you suggest a set of vendor and model strings?